airflow/migrations/versions/92c57b58940d_add_fab_tables.py

```diff
@@ -0,0 +1,173 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Create FAB Tables & Increase length of ab_view_menu.name column
```

Comment:
This should be two migrations please

Reply:
Sure

Reply:
Done in https://github.com/apache/airflow/pull/12352/commits/c4d4c78951228b0ec7a598b151fa0142e6191a3e